docs/stories/stories.md

```diff
@@ -0,0 +1,79 @@
+# COGS User Stories
+
+## Roles:
+User
+Project owner
+Principal Investigator
```

Comment:
We do not force this to be a user, so is a free-form string which *could* point at the Tech Lead, but might not.  In mind here, we have situations where PI is perhaps very high-level or rather less technically engaged than their researchers.